include/APIClient/Router.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+#include <map>
+
+namespace APIClient {
+
+namespace Router {
+
+    enum class Route : uint8_t {
+        getAll,
+        getVulkanInfoWithId,
```

Comment:
You have changed that but it's not obvious what you are doing on this route.
Change it to `getGpu`.